Console/Command/RestoreUseDefaultValueCommand.php

```diff
@@ -35,7 +36,8 @@ protected function configure()
             ->setDescription("
                 Restore product's 'Use Default Value' if the non-global value is the same as the global value
             ")
-            ->addOption('dry-run');
+            ->addOption('dry-run')
+            ->addOption('entity', null, InputOption::VALUE_REQUIRED, "Set entity to cleanup (product or category)");
```

Comment:
```suggestion
            ->addOption(
                'entity',
                null,
                InputOption::VALUE_OPTIONAL,
                'Set entity to cleanup (product or category)',
                'product'
            );
```

I think we should keep the current behaviour as the default and only add this as an additional option.